src/frameworks/fastify.js

```diff
@@ -1,7 +1,14 @@
 const fp = require('fastify-plugin');
 
 function getValidator(validateRequest) {
-    return () => {
+    const urijs = require('uri-js');
```

Comment:
you've declared the `uri-js` is optional, but always requiring and using the package

Reply:
But it's only used if you use fastify, hence it's optional for everyone else.

Reply:
Made error more explicit for when it fails due to missing dependency. I can move it from peerDependency to just a dependency if you don't mind having a new dependency, this will reduce amount of effort needed from user side.

Reply:
it's ok like that for now 👍